packages/logs/lib/models/helpers.ts

```diff
@@ -124,7 +124,7 @@ export const operationTypeToMessage: Record<ConcatOperationList, string> = {
     'action:run': 'Action execution',
     'admin:impersonation': 'Admin logged into another account',
     'auth:create_connection': 'Create connection',
-    'auth:post_connection': 'Post connection script execution',
+    'auth:post_connection': 'post connection execution',
```

Comment:
not requiring changes: I just noticed both `post-connection` and `onConnectionCreated` are using `auth:post_connection` maybe it would be better to dissociate them, e.g: `internal_on_connection_created` and `external_on_connection_created` or clearly identify them if possible

Also one question, why not move both into the `on` folder?

Reply:
I am planning to change the log types once I introduced the new event. For now they are still post connection related so I think it is fine.
I also want to keep the internal post connection separate from the rest because they are technically very different. They are not schedule custom scripts executed by the runner but some provider specific piece of logic executed by server